test/common/wpt.js

```diff
@@ -496,6 +498,9 @@ class WPTRunner {
       this.resultCallback(filename, { status: 2, name: 'Unknown' });
     }
     this.inProgress.delete(filename);
+    // Always force termination of the worker. Some tests allocate resources
+    // that would otherwise keep it alive.
+    this.workers.get(filename).terminate();
```

Comment:
If it's safe to ignore the Promise status, maybe add an empty catch handler?
```suggestion
    this.workers.get(filename).terminate().catch(() => {});
```

Reply:
This promise can never reject
https://github.com/nodejs/node/blob/f3d3769f9f6da29fb562f25074cda6c86a0366df/lib/internal/worker.js#L344-L354